src/modules/codes/RedeemableCodes.ts

```diff
@@ -190,6 +190,7 @@ export default class RedeemableCodes implements Saveable {
                 return refund;
             }),
             new RedeemableCode('tutorial-skip', -253994129, false, async () => {
+                App.game.keyItems.gainKeyItem(KeyItemType.Dungeon_ticket, true);
```

Comment:
This should be moved to after the `gainDungeonTokens` line. Can you also update the notification text to indicate it was given to the player along with skipping the tutorial and finding some dungeon tokens?